### DISCUSSION TITLE
How do I add new song free play?

### DISCUSSION THREAD

**Discussion by Reyororo** [KadeArchive/Kade-Engine#1671] [Q&A] [answered]
How do I add new song free play?

**Comment by napstaa967** [comment, marked as answer]
preload/data
open freeplaySongList.txt
add a new line and type the following in: (replace the stuff inside <> with the song):

`<Song Name>:<Song Icon (char)>:<Week Number>`

**Reply by Reyororo** [reply]
> preload/data
> open freeplaySongList.txt
> add a new line and type the following in: (replace the stuff inside <> with the song):
> 
> `<Song Name>:<Song Icon (char)>:<Week Number>`

How can I find the week number?

**Reply by napstaa967** [reply]
> > preload/data
> > open freeplaySongList.txt
> > add a new line and type the following in: (replace the stuff inside <> with the song):
> > `<Song Name>:<Song Icon (char)>:<Week Number>`
> 
> How can I find the week number?

its basically just the number of the week of the song

if you don't know it just set it to 0 or 1

**Reply by napstaa967** [reply]
aka lets say you wanna add fresh

you would set the number to 1
if you want to add roses, then its 6

if you added a custom 7th week and wanna add its songs to freplay just set the number to 7

**Reply by Reyororo** [reply]
> aka lets say you wanna add fresh
> 
> you would set the number to 1
> if you want to add roses, then its 6
> 
> if you added a custom 7th week and wanna add its songs to freplay just set the number to 7

Thank you
